src/core/Akka/Routing/RouterConfig.cs

```diff
@@ -129,9 +129,37 @@ public override bool Equals(RouterConfig other)
     }
 
 
-    //TODO: ensure this can be serialized/deserialized fully
-    public abstract class Pool : RouterConfig
+    //TODO: ensure this can be serialized/deserialized fully    
+    public abstract class Pool : RouterConfig, IEquatable<Pool>
     {
+        //TODO: add supervisor strategy to the equality compare
+        public bool Equals(Pool other)
```

Comment:
Let's rename the "overrides" region to "equality" and move all of these members inside of it - just for organization purposes.